conans/model/build_info.py

```diff
@@ -334,124 +334,121 @@ def get_property(self, property_name):
             pass
 
     def get_init(self, attribute, default):
+        # Similar to dict.setdefault
```

Comment:
Not for now, but we should change it to `setdefault` for instance. `get_init` sounds weird, wdyt?